.github/workflows/release.yml

```diff
@@ -0,0 +1,12 @@
+name: Release
+on:
+  push:
+    branches:
+      - main
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v4
+        with:
+          token: ${{secrets.GITHUB_TOKEN}}
```

Comment:
:thinking: does the token by default have permissions to raise PRs etc.? The basic example (https://github.com/googleapis/release-please-action?tab=readme-ov-file#basic-configuration) uses:

```yaml

permissions:
  contents: write
  pull-requests: write
```

(but then goes on to use `${{ secrets.MY_RELEASE_PLEASE_TOKEN }}` which makes those permissions unused anyway :upside_down_face:)

But in `uwos-go` we never set these, so I guess not?

Reply:
Yeah, I think if this org-wide secret is good enough for uwos-go, it should be good enough for shared-kustomize-bases :slightly_smiling_face:

Reply:
> Yeah, I think if this org-wide secret is good enough for uwos-go, it should be good enough for shared-kustomize-bases 🙂

It's not an org-wide secret, it's the one GitHub gives us in each action run, but looking at some docs https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token I assume our org has things setup in "Default access(permissive)" mode so we get the permissions we need by default